activesupport/lib/active_support/encrypted_file.rb

```diff
@@ -26,18 +26,21 @@ def self.generate_key
     end
 
 
-    attr_reader :content_path, :key_path, :env_key
+    attr_reader :content_path, :key_path, :env_key, :raise_if_missing_key
 
-    def initialize(content_path:, key_path:, env_key:)
+    def initialize(content_path:, key_path:, env_key:, raise_if_missing_key:)
       @content_path, @key_path = Pathname.new(content_path), Pathname.new(key_path)
       @env_key = env_key
```

Comment:
Also preferred the `@env_key, @raise_if_missing_key = …` assign. The extra array allocation ain't really an appropriate concern at this level. People aren't likely to instantiate a ton of instances of this class, so 1 extra array is a drop in a bucket.